templates/common/alibabacloud/files/usr-local-lib-systemd-system-generators-alibaba-kubelet-extra-env-generator.sh.yaml

```diff
@@ -0,0 +1,29 @@
+mode: 0755
+path: "/usr/local/lib/systemd/system-generators/alibaba-kubelet-extra-env-generator.sh"
+contents:
+  inline: |
+    #!/bin/sh
+    set -e -o pipefail
```

Comment:
Ditto here re strict mode.  

Also note https://github.com/coreos/fedora-coreos-config/pull/1797/commits/1a204418d187834453dea01d991b3c3353937b98
specifically https://github.com/coreos/fedora-coreos-config/blob/testing-devel/overlay.d/05core/usr/lib/coreos/generator-lib.sh#L3
to ensure we get log messages if something goes wrong in this code.